docs/reference/localization.md

```diff
@@ -2,193 +2,256 @@
+  * Any strings in the `.po` files that no longer have entries in the `.pot` files are commented out.
+
+The translations in `.po` (Portable Object) files are stored with a relatively simple format. Here is an example:
+
+```po
+#: .es5/views/confirm_reset_password.js:195
+msgid "Password reset"
+msgstr "Slaptažodžio atkūrimas"
+```
+
+- First line (#: .es5…): location(s) where the string is used in the source code.
+- Second line (msgid): string in English.
+- Third line (msgstr): the translation (if available). If no translation is provided, msgstr is an empty string `""`
+
+# Localization process
+* Localizers are notified that Mozilla accounts strings have changed.
+* Localizers make updates and a review request is issued.
+* Reviewer accepts changes.
+* Changes are pushed back to the [fxa-content-server-l10n][#l10n-repo] repo as a commit to master.
+  * Every commit to master causes a [linting job][#grunt-lint-l10n] to be run.
+    * The linter checks to ensure:
+      * All HTML within the translations is well formed, e.g., no unclosed anchor tags.
+      * No unexpected HTML elements, attributes or values exist.
+      * All named variables in the translations exist in the English strings.
+    * These checks prevent:
+      * l10n being used as an attack vector since HTML element attributes can contain JavaScript or external links.
+      * Incorrect page renderings due to malformed HTML in l10n strings.
+
+# Build process
+
+The latest strings from [fxa-content-server-l10n repo][#download-l10n-script] are downloaded and bundled with each build. Newly localized strings from Pontoon are not available (on stage/production) until a new release is created.
+
+* On prebuild, [l10n:prime][#l10n-prime-script] pulls the latest localization files into target workspaces.
+* The build process calls a `build-l10n` task:
+    - In fxa-content-server:
+        - The [l10n-create-json grunt task][#l10n-create-json-grunt-task] copies all translated `.po` files from the fxa-content-server-l10n repo and runs the [grunt po2json task][#grunt-po2json-task], once for each of the 2 `.po` files for each locale, resulting in 2 `.json` files for each locale.
+        - A copy of the compiled JavaScript bundles is created for each locale.
+        - Static JavaScript resources are deployed to a 3rd party CDN (Amazon) and [Subresouce Integrity (SRI)][#sri] is used to ensure the resources are not modified on the CDN. Each JavaScript bundle (per locale) has its own SRI hash.
+    - In other localized packages (fxa-auth-server, fxa-payments, fxa-react, fxa-settings, fxa-shared):
+        - Individual source `en.ftl` files for the default 'en' locale are merged into a package-level `.ftl` file (`merge-ftl` grunt task) to ensure local builds have access to the latest FTL strings.
+        - l10n bundles are created for use by Fluent's `<LocalizationProvider>` with [bundle.sh][#l10n-bundle-script]. 
+
+# Additional notes
```

Comment:
Here are a few more that might be good to note somewhere:

* If we forget to update the FTL ID when we update the FTL string and fallback text, it's a "silent change" on the pontoon side. The source language would change, but all other languages would not know it changed.
* If updates are made on the fxa-content-server-l10n repo side and sync back to Pontoon, they come in as "by imports"
* Should we have a note about grouping comments in FTL? I noted [this](https://github.com/mozilla/fxa/pull/14815#discussion_r1091268478)